api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -511,6 +511,7 @@ public void test_DuccBecomesOutdated() {
     // bypass the other access requirements
     final DbUser dbUser = userDao.findUserByUserId(userId);
     accessModuleService.updateCompletionTime(dbUser, AccessModuleName.ERA_COMMONS, TIMESTAMP);
+    accessModuleService.updateCompletionTime(dbUser, AccessModuleName.RAS_LOGIN_GOV, TIMESTAMP);
```

Comment:
why is this the only test which needs to be changed?  I would expect many tests to fail

Reply:
seems that is the only test that checks user tier.

Reply:
I was thinking of UserServiceAccessTest - but I see now that this flag is already true there.  👍